lisa/tools/mv.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+
+
+class Mv(Tool):
```

Comment:
There are some use of "mv" command directly. Can you please replace those calls with this tool?

Reply:
Done.